pyds8k/resources/ds8k/v1/common/mixins.py

```diff
@@ -600,10 +629,14 @@ def get_flashcopy(self, fcid=None):
         if not self.id:
             raise IDMissingError()
         if fcid:
-            return self.one(types.DS8K_FLASHCOPY, fcid).get()
-        flashcopies = self.all(types.DS8K_FLASHCOPY).list()
+            return self.one('{}.{}'.format(
```

Comment:
Please create get_cs_flashcopies function to contain these modification

Reply:
done